fixed Typos/mistakes

robin025 opened the pull request:
fixes some typos

robin025 commented:
Also Fixed Some typos and mistakes

GeorgLink commented on About/About/faq.md:
```diff
@@ -1,14 +1,13 @@
 # FAQ
 
-## How do open source community health metrics help me understand whether an open source project is healthy?
+# How do open source community health metrics help me understand whether an open source project is healthy?
```
The questions should continue to be a sub-heading with `##`.

robin025 replied:
Done...I made the changes as suggested in the review!!

GeorgLink replied:
Thank you!

GeorgLink reviewed (changes requested):
Thanks @robin025!

Please leave the FAQ questions as sub-headings.

I was thinking that we don't need to change the past CHAOSScast event pages, but I don't feel too strongly about that.

GeorgLink commented on release/release-notes.md:
```diff
@@ -1,8 +1,8 @@
 # Release History
-CHAOSS metrics are released continuously. The regular release is when we update the version number, update the full release notes, and make a big announcement. These releases occur one to two times a year and may correspond with the dates of a CHAOSScon event. Prior to regular release, continuous released metrics go through a comment period of at least 30 days.
+CHAOSS metrics are released continuously. The regular release is when we update the version number, update the full release notes, and make a big announcement. These releases occur one to two times a year and may correspond with the dates of a CHAOSScon event. Before regular release, continuously released metrics go through a comment period of at least 30 days.
```
Please undo this change

GeorgLink commented on release/metrics-head.md:
```diff
@@ -10,4 +10,4 @@ _To make suggestions or edits to this webpage please visit the [website repo](ht
 Metrics on this page were debated in working groups and undergo a 30 day comment period to ensure validity. Released metrics are only a subset of many possible metrics. CHAOSS acknowledges that more metrics exist and is working to identify and release new metrics in the future. If you would like to learn more about metrics, suggest new metrics, and or help define metrics please visit our [partici ...
 
-The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the CHAOSS community. **Ethical challenges** exist around protecting community members and empowering them with their personal information. **Legal challenges** exist around GDPR and similar laws or regulations that protect personal information of community members. Particular ...
+The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the CHAOSS community. **Ethical challenges** exist around protecting community members and empowering them with their personal information. **Legal challenges** exist around GDPR and similar laws or regulations that protect the personal information of community members. Partic ...
```
Please undo this change

GeorgLink commented on Programs/DIBadging/description.md:
```diff
@@ -1,5 +1,5 @@
 # Diversity & Inclusion Event Badging Form
-Thank you for showing interest in D&I Badging for your event! The Event Badging section of CHAOSS Badging is about measuring inclusivity of different technical events through human reviews.
+Thank you for showing interest in D&I Badging for your event! The Event Badging section of CHAOSS Badging is about measuring the inclusivity of different technical events through human reviews.
```
Please undo this change

GeorgLink commented on CHAOSScon/2020-Shanghai-meetup/speakers.md:
```diff
@@ -71,7 +71,7 @@ Participated in the open source community for more than 6 years. Currently, she
 _Director of Sales - Bitergia_
 
-Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enj ...
+Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years of experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg  ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2020-Shanghai-meetup/speakers.md:
```diff
@@ -55,7 +55,7 @@ Xiaoya is currently a postgraduate student of ECNU, major in data science and so
 _Co-Founder - Bitergia, InnerSource Commons_
 
-Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests about open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wi ...
+Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests in open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikim ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2020-Shanghai-meetup/speakers.md:
```diff
@@ -22,7 +22,7 @@ Wang Wei, researcher and doctoral supervisor of the School of Data Science and E
 _Head of Community - Cube Dev_
 
-Ray is the Head of Community at Cube Dev where he is helping to grow the community of contributors to cube.js. Prior to Cube Dev, Ray managed open source communities at GitLab and the Linux Foundation. He has over 15 years of experience in the high-tech industry in roles ranging from software engineer, product manager, program manager, and team lead at companies such as EDS, Intel, and Medallia.  ...
+Ray is the Head of Community at Cube Dev where he is helping to grow the community of contributors to cube.js. Prior to Cube Dev, Ray managed open source communities at GitLab and the Linux Foundation. He has over 15 years of experience in the high-tech industry in roles ranging from a softwaresoftware engineer, product manager, program manager, and team lead at companies such as EDS, Intel, and  ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2020-Shanghai-meetup/about.md:
```diff
@@ -1,6 +1,6 @@
 ## About CHAOSS Shanghai Meetup
 
-Meet the CHAOSS community. Learn about metrics and tools used by several open source projects, communities, and engineering teams to track and analyze their development activities, communities health, diversity, risk, and value.
+Meet the CHAOSS community. Learn about metrics and tools used by several open source projects, communities, and engineering teams to track and analyze their development activities, community's health, diversity, risk, and value.
```
Please undo this change

GeorgLink commented on CHAOSScon/2019NA/speakers.md:
```diff
@@ -93,7 +93,7 @@ Dawn works on OSS Strategy at Pivotal in London. She has more than 20 years of e
 _Director of Sales - Bitergia_
 
-Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enj ...
+Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years of experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg  ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2019NA/speakers.md:
```diff
@@ -24,9 +24,9 @@ _PhD Cand. / DevOps - Polytechnique Montreal_
+I am focused on empirical research on ecosystems software releases.  I apply mixed method research as a means to answer both the how and why in my findings, which is beneficial to both the academic and industrial communities.  I am Using a DevOps approach with machine learning to make crucial decisions. Currently, I am involved with the release team of OpenStack and serving as the PC chair of AI  ...
 Before my current experience, I had Worked on interesting software engineering topics for swarms of robots. Moreover, my expertise in cloud computing is an asset, which I bring along. In my thesis, I investigated the phenomena of live migrations and how we can apply both the supervised and reinforced learning technique (Machine learning algorithms) on live migration in the cloud.  
-I was part of the team; team lead on an exciting project at LASSENA Laboratory Montreal, Canada. My main task includes building test bench simulator for micro iBB black boxes for cars in other to reconstruct accident scenario to understand driver's behavior.
+I was part of the team; team lead on an exciting project at LASSENA Laboratory Montreal, Canada. My main task includes building test bench simulator for micro iBB black boxes for cars in other to reconstruct accident scenarios to understand driver's behavior.
```
Please undo this change

GeorgLink commented on CHAOSScon/2019NA/speakers.md:
```diff
@@ -24,9 +24,9 @@ _PhD Cand. / DevOps - Polytechnique Montreal_
 Ph.D. Candidate.,   Software Engineering, MCIS -- Laboratory, École Polytechnique de Montréal.
 OpenStack Member Level: Foundation Member.
-I am focused on empirical research on ecosystems software releases.  I apply mixed method research as a means to answer both the how and why in my findings, which is beneficiary to both the academic and industrial communities.  I am Using a DevOps approach with machine learning to make crucial decisions. Currently, I am involved with the release team of OpenStack and serving as the PC chair of AI ...
+I am focused on empirical research on ecosystems software releases.  I apply mixed method research as a means to answer both the how and why in my findings, which is beneficial to both the academic and industrial communities.  I am Using a DevOps approach with machine learning to make crucial decisions. Currently, I am involved with the release team of OpenStack and serving as the PC chair of AI  ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2019EU/speakers.md:
```diff
@@ -63,11 +63,11 @@ Session: **A Tale of Metrics Faux Pas: Answers Without Questions**
 _Data Analyst - Bitergia_
 
-Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests about open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wi ...
+Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests in open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikim ...
```
Please undo this change

GeorgLink commented on CHAOSScon/2019EU/speakers.md:
```diff
@@ -37,11 +37,11 @@ Session: **Your Team’s Open Source Contributions, in One Dashboard**
 _Marketing Specialist - Bitergia_
 
-Ana holds a Bachelor degree in Marketing and is currently working at Bitergia, a software development analytics company focused on OpenSource and InnerSource environments.
+Ana holds a Bachelor's degree in Marketing and is currently working at Bitergia, a software development analytics company focused on OpenSource and InnerSource environments.
```
Please undo this change

GeorgLink commented on CHAOSScon/2019EU/speakers.md:
```diff
@@ -25,10 +25,10 @@ Session: **Creating a Collection of Panels**
 _Open Source Engineer - VMware_
 
-Alex is an Open Source Engineer at VMware Open Source Technology Center, where he works on VMware open source contribution analytics. Before VMware, Alex was a getting his masters in Software Engineering at Polytechnique Montreal, where he researched collaboration and contribution in Open Source communities.
+Alex is an Open Source Engineer at VMware Open Source Technology Center, where he works on VMware open source contribution analytics. Before VMware, Alex was getting his master's in Software Engineering at Polytechnique Montreal, where he researched collaboration and contribution in Open Source communities.
```
Please undo this change

GeorgLink commented on CHAOSScon/2019EU/speakers.md:
```diff
@@ -6,7 +6,7 @@
 _Researcher and Software Developer - Bitergia_
 
-Software engineer at Bitergia and analytics team member. After finishing his PhD on web page representation for clustering tasks in 2012, he has been working in projects involving data retrieval and analysis from different data sources. He has experience with Java, Spring, Python, MySQL, Apache Solr and ElasticSearch, among others. Apart from implementing metrics with Python and Kibana and accord ...
+Software engineer at Bitergia and analytics team member. After finishing his PhD on web page representation for clustering tasks in 2012, he has been working on projects involving data retrieval and analysis from different data sources. He has experience with Java, Spring, Python, MySQL, Apache Solr and ElasticSearch, among others. Apart from implementing metrics with Python and Kibana and accord ...
```
Please undo this change

GeorgLink reviewed (changes requested):
Please undo changes to personal bios of CHAOSScon speakers. I want to respect the bios they sent and not alter them (especially not after the event is done).

GeorgLink reviewed (changes requested):
Please remove the alumni.md changes from this PR and create a separate PR.

It is best practice to have small PRs so that the reviewer can focus on specific changes at a time.

The spelling changes are unrelated to the refactoring of the alumni.md page -- so these should be two separate PRs.

Thanks you!

robin025 commented:
> Thanks @robin025!
> 
> Please leave the FAQ questions as sub-headings.
> 
> I was thinking that we don't need to change the past CHAOSScast event pages, but I don't feel too strongly about that.

@GeorgLink  the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it

GeorgLink commented:
> @GeorgLink the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it

Okay to fix links and typos. But let's leave the speaker bios untouched if it's stylistic issues.

robin025 commented:
> > @GeorgLink the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it
> 
> Okay to fix links and typos. But let's leave the speaker bios untouched if it's stylistic issues.

Done @GeorgLink reverted changes from speaker bios

GeorgLink commented:
Thanks @robin025 

Would you be comfortable to squash the commits and rebase them to the current master branch?
If not, that's fine and we can work around it.

robin025 commented:
> Thanks @robin025
> 
> Would you be comfortable to squash the commits and rebase them to the current master branch?
> If not, that's fine and we can work around it.

@GeorgLink  Done.... Squashed all the commits and rebased it to master branch

GeorgLink commented:
superseded by #590
